blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java

```diff
@@ -49,13 +47,11 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import static org.jboss.netty.channel.Channels.succeededFuture;
-import static org.jboss.netty.channel.Channels.write;
-import static org.jboss.netty.handler.codec.http.HttpHeaders.Names.*;
-import static org.jboss.netty.handler.codec.http.HttpResponseStatus.*;
-import static org.jboss.netty.handler.codec.http.HttpVersion.HTTP_1_1;
+import static io.netty.handler.codec.http.HttpResponseStatus.*;
```

Comment:
If we want to avoid wildcard import then I think we should change the `idea.xml` on master first.